BookingsApi/BookingsApi.IntegrationTests/Database/Queries/GetAllocationHearingsBySearchQueryTests.cs

```diff
@@ -25,8 +29,10 @@ public class GetAllocationHearingsBySearchQueryTests : DatabaseTestsBase
     [SetUp]
     public async Task Setup()
     {
+        var settings = new SettingsConfiguration {IsTest = true};
+        var options = Options.Create(settings);
         _context = new BookingsDbContext(BookingsDbContextOptions);
-        _handler = new GetAllocationHearingsBySearchQueryHandler(_context, isTest: true);
+        _handler = new GetAllocationHearingsBySearchQueryHandler(_context, options);
```

Comment:
similar to my previous comment to avoid passing in a boolean value only for test purpose